staging_service/app.py

```diff
@@ -42,6 +42,7 @@
 VERSION = "1.3.6"
 
 _DATATYPE_MAPPINGS = None
+_DTS_MANIFEST_SCHEMA = None
```

Comment:
The pattern seems to be making strings like this global, so I'm continuing that.